src/js/media-container.ts

```diff
@@ -685,6 +700,14 @@ class MediaContainer extends globalThis.HTMLElement {
     setBooleanAttr(this, Attributes.NO_AUTOHIDE, value);
   }
 
+  get alwaysHideControls(): boolean | undefined {
```

Comment:
let's discuss this in open source office hours.

we have the attribute `autohide` which is a number in seconds and -1 turns it off.

there is also the `noautohide` attribute which can be added to controls to not get automatically hidden.